<launch>

	<arg name="rviz" default="true" />

	<rosparam command="load" file="$(find vio_in_lidar_map)/config/NTU_VIRAL.yaml" />

	<node pkg="vio_in_lidar_map" type="vio_in_lidar_map" name="vio_in_lidar_map" output="screen" >
		<rosparam file="$(find vio_in_lidar_map)/config/camera_NTU_VIRAL.yaml" />
	</node>
	
	<group if="$(arg rviz)">
		<node launch-prefix="nice" pkg="rviz" type="rviz" name="rviz" args="-d $(find vio_in_lidar_map)/rviz_cfg/loam_livox_ouster.rviz" />
	</group>
	<!-- <node pkg="image_transport" type="republish" name="republish" args="compressed in:=/left_camera/image raw out:=/left_camera/image" output="screen" respawn="true"/> -->

launch-prefix="gdb -ex run --args"
</launch> 	
